packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1281,6 +1282,14 @@ contract Validators is
     slashingMultiplierResetPeriod = value;
   }
 
+  /**
+   * @notice Sets the downtimeGracePeriod property if called by owner.
+   * @param value New downtime gracec period for calculating epoch scores.
+   */
+  function setDowntimeGracePeriod(uint256 value) public nonReentrant onlyOwner {
+    downtimeGracePeriod = value;
```

Comment:
We'd probably want to add some bounds here? I.e. is there a reasonable upper limit we can put in place here? Something like the epoch length?

Reply:
Agreed. However it has to come via Governance which is a reasonable stop gap. Happy to put it in after this is merged